pkg/antctl/raw/set/flowaggregator/command_test.go

```diff
@@ -0,0 +1,199 @@
+    server: https://localhost
+  name: fake-cluster
+contexts:
+- context:
+    cluster:  fake-cluster
+    user:  user-id
+  name:  fake-cluster
+current-context:  fake-cluster
+kind: Config`),
+		},
+	}
+
+	for _, tc := range tcs {
+		fakeKubeconfig, err := os.CreateTemp("", "fakeKubeconfig")
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer os.Remove(fakeKubeconfig.Name())
+		fakeKubeconfig.Write(tc.fakeConfigs)
+		if cmd.Flag("kubeconfig") != nil {
+			cmd.Flag("kubeconfig").Value.Set(fakeKubeconfig.Name())
+		} else {
+			cmd.Flags().String("kubeconfig", fakeKubeconfig.Name(), "path of kubeconfig")
+		}
+		err = updateRunE(cmd, args)
+		assert.Error(t, err)
+	}
+}
+
+func TestSetBoolOrFail(t *testing.T) {
+	tcs := []struct {
+		b             bool
+		value         string
+		errorExpected bool
+	}{
+		{
+			value:         "1",
+			errorExpected: false,
+		},
+		{
+			value:         "tt",
+			errorExpected: true,
+		},
+	}
+
+	for _, tc := range tcs {
+		err := setBoolOrFail(&tc.b, tc.value)
+		if tc.errorExpected {
+			assert.Error(t, err)
+		} else {
+			assert.NoError(t, err)
+		}
+	}
+}
+
+func TestSetStringOrFail(t *testing.T) {
+	tc := []struct {
+		b     string
+		value string
+	}{
+		{
+			value: "11mm",
+		},
+	}
+
+	_ = setStringOrFail(&tc[0].b, tc[0].value)
+	assert.Equal(t, tc[0].b, tc[0].value)
+}
+
+func TestSetCommitIntervalOrFail(t *testing.T) {
+	tcs := []struct {
+		b             string
+		value         string
+		errorExpected bool
+	}{
+		{
+			value:         "1m25s",
+			errorExpected: false,
+		},
+		{
+			value:         "0.5s",
+			errorExpected: true,
+		},
+		{
+			value:         "0.5ss",
+			errorExpected: true,
+		},
+	}
+
+	for _, tc := range tcs {
+		err := setCommitIntervalOrFail(&tc.b, tc.value)
+		if tc.errorExpected {
+			assert.Error(t, err)
+		} else {
+			assert.Equal(t, tc.b, tc.value)
+		}
+	}
+}
+
+func TestGetFAConfigMap(t *testing.T) {
+	k8sClient := testclient.NewSimpleClientset()
+	tcs := []struct {
+		faConfigMap   *corev1.ConfigMap
+		errorExpected bool
+	}{
+		{
+			faConfigMap: &corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "testConfigMap",
+					Namespace: "flow-aggregator",
+				},
+			},
+			errorExpected: false,
+		},
+		{
+			faConfigMap: &corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "",
+					Namespace: "flow-aggregator",
+				},
+			},
+			errorExpected: true,
+		},
+	}
+
+	for _, tc := range tcs {
```

Comment:
You may take other test files as reference. A better way to use table-driven test is provide a name field to it, so you can locate the case quicker when there is a failure. 
```
func TestGetFAConfigMap(t *testing.T) {
	existingCM := &corev1.ConfigMap{
		ObjectMeta: metav1.ObjectMeta{
			Name:      "flow-aggregator",
			Namespace: "flow-aggregator",
		},
	}
	k8sClient := fake.NewSimpleClientset(existingCM)

	tests := []struct {
		name        string
		expectedErr string
		configName  string
	}{
		{
			name:       "get ConfigMap successfully",
			configName: "flow-aggregator",
		},
		{
			name:        "emtpy ConfigMap name",
			expectedErr: "failed to locate flow-aggregator-config ConfigMap volume",
		},
		{
			name:        "fail to get ConfigMap",
			expectedErr: "failed to get ConfigMap",
			configName:  "non-existing-cm",
		},
	}
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
			cm, err := GetFAConfigMap(k8sClient, tt.configName)
			if err != nil {
				assert.Contains(t, err.Error(), tt.expectedErr)
			} else {
				assert.Equal(t, existingCM, cm)
			}
		})
	}
}
```